openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -443,25 +444,27 @@ def post(self, request):
                 retire_dot_oauth2_models(request.user)
                 AccountRecovery.retire_recovery_email(request.user.id)
 
-                try:
-                    # Send notification email to user
-                    site = Site.objects.get_current()
-                    notification_context = get_base_template_context(site)
-                    notification_context.update({'full_name': request.user.profile.name})
-                    language_code = request.user.preferences.model.get_value(
-                        request.user,
-                        LANGUAGE_KEY,
-                        default=settings.LANGUAGE_CODE
-                    )
-                    notification = DeletionNotificationMessage().personalize(
-                        recipient=Recipient(username='', email_address=user_email),
-                        language=language_code,
-                        user_context=notification_context,
-                    )
-                    ace.send(notification)
-                except Exception as exc:
-                    log.exception('Error sending out deletion notification email')
-                    raise
+                if is_testing_environment():
```

Comment:
Is there any specific requirement to disable notifications? If not then we should not change the default flow.

Reply:
I discussed this with Imran bhai and we decided we didn't need any for now.